package.json

```diff
@@ -4,13 +4,18 @@
   "description": "",
   "main": "app.js",
   "dependencies": {
+    "@babel/core": "^7.11.5",
+    "@babel/node": "^7.10.5",
+    "@babel/preset-env": "^7.11.5",
```

Comment:
@babel/core @babel/node @babel/preset-env should be under devDependencies, you need to fix this